### PR TITLE
Add timeZones in settings and update Accounts tab

### LINKED ISSUE #1233: Implement Account Tab in Settings
Account Info should be as in screenshot below.

Ensure this data is stored on the server backend.

screenshot from 2018-05-25 01-04-01

### PR DESCRIPTION
**Pull request by PrP-11**
Fixes #1233 

Changes: Implemented TimeZones in accounts. Users can now set their timezones and default language , this data is updated on server in real time. 

Demo Link: http://ugliest-camp.surge.sh

Screenshots for the change: 
settings

Timezone select menu is implemented as a searchable menu. Users can search by typing a single character.

settings

### CODE REVIEW

**Comment by PrP-11** [comment]
@akshatnitd codacy is failing because it says use double quotes for strings. But the code follows single quotes also if I use double quotes then travis is failing.

**Comment by AnupKumarPanwar** [comment]
@akshatnitd  I agree with @PrP-11. Travis and Codacy sometimes conflict between their standards.

**Comment by akshatnitd** [comment]
Yes, the mentors can have a final take on it!

File: src/components/ChatApp/Settings/Settings.react.js [hunk 1]
```diff
@@ -597,7 +609,7 @@ class Settings extends Component {
 		}
 		else{
 			this.setState({
-			selectedSetting: cookies.get('loggedIn')? 'Account':'ChatApp Settings'
+			selectedSetting: cookies.get('loggedIn')? 'General':'ChatApp Settings'
```
Comment (mariobehling) [inline src/components/ChatApp/Settings/Settings.react.js]: I missed this change. Where was that change part of the issue? Why was that change? Please revert.

**Comment by mariobehling** [comment]
I missed changes here and I should not have merged this. Please stick to what is mentioned in issues and pull requests and if you change anything else add this into the description to point reviewers to it. Please revert "general" to "account".

**Comment by PrP-11** [comment]
@mariobehling I thought General would be more preferable here. I will revert the changes.

**Pull request merged**